client/src/components/SendMessage.js

```diff
@@ -36,36 +39,73 @@ class SendMessage extends React.Component {
     return;
   }
 
+  hideEmojiPicker() {
+      this.setState({
+          ...this.state,
+          emojiPickerActive : false
+      });
+  }
+  onKeyUp(event) {
```

Comment:
With the refactoring from all the feedback, we can get rid of onKeyDown completely.